src/data/partnersData.tsx

```diff
@@ -0,0 +1,43 @@
+import Translate from "@docusaurus/Translate";
+import React from "react";
+
+import huawei from "../../static/img/supporters/huawei.png";
+import daocloud from "../../static/img/supporters/DaoCloud.png";
+import DSK from "../../static/img/supporters/DSK.png";
+
+const PartnersList = [
+  {
+    image: huawei,
+    title: "Huawei",
+    webURL: "https://www.huaweicloud.com/intl/en-us/",
```

Comment:
```suggestion
    webURL: "https://www.huaweicloud.com/intl/en-us/product/ucs.html",
```
I guess a service URL might be more helpful to users.